aws/components/topic/setup.ftl

```diff
@@ -247,6 +247,28 @@
                         /]
                     [/#if]
                 [/#list]
+
+                [#list solution.Endpoints as id, endpoint]
+                    [#switch endpoint.Protocol ]
+                        [#case "http"]
+                        [#case "https" ]
+                            [#local deliveryPolicy = getSNSDeliveryPolicy(solution.DeliveryPolicy) ]
```

Comment:
Think this needs to be initialised as part of the loop for the case of the endpoint not being http/s.

Reply:
Ta it was falling back to the one defined for the links